src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -288,12 +842,16 @@ def map_keyvalue_to_featureflagvalue(keyvalue):
             FeatureFlagConstants.ID,
             FeatureFlagConstants.DESCRIPTION,
             FeatureFlagConstants.ENABLED,
-            FeatureFlagConstants.CONDITIONS}
+            FeatureFlagConstants.CONDITIONS,
+        }
         if valid_fields != feature_flag_dict.keys():
```

Comment:
I would advise we change this to rather ensure that all the fields are in the feature flag dictionary and not that the two are exactly equal. We could break CLI if user adds a field we consider unknown.

How about we find the intersection of the expected and received fields and confirm that this is equal to the expected fields?